packages/schema-type/src/solidity/SchemaType.sol

```diff
@@ -211,27 +211,20 @@ enum SchemaType {
  * Get the length of the data for the given schema type
  * (Because Solidity doesn't support constant arrays, we need to use a function)
  */
-function getStaticByteLength(SchemaType schemaType) pure returns (uint256) {
+function getStaticByteLength(SchemaType schemaType) pure returns (uint256 r) {
```

Comment:
for free functions, do we not need/get any benefit from `internal`? (I assume not)

Reply:
free functions can't have visibility modifiers at all
effectively they're all always internal